src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/Runtime/BindingFlagSupport/PropertyPolicies.cs

```diff
@@ -99,5 +99,35 @@ public sealed override bool OkToIgnoreAmbiguity(PropertyInfo m1, PropertyInfo m2
 
             return accessor;
         }
+
+        private static MethodInfo? GetMostAccessibleAccessor(PropertyInfo property)
+        {
+            MethodInfo? getter = property.GetMethod;
+            MethodInfo? setter = property.SetMethod;
+
+            if (getter == null)
+                return setter;
+            if (setter == null)
+                return getter;
+
+            // Return the setter if it's more accessible, otherwise return the getter (preserving original behavior when equal)
```

Comment:
Is `original behavior` the "behavior before this PR"? If so, it's not a very useful comment.